components/Org/OrgTabs.messages.js

```diff
@@ -0,0 +1,53 @@
+import { FormattedMessage } from 'react-intl'
+import React from 'react'
+
+const orgTab =
+  <FormattedMessage
+    id='orgTabs.about'
+    defaultMessage='About'
+    description='Tab label on OrgDetailsPage'
+  />
+
+const orgMemberTab =
+  <FormattedMessage
+    id='orgMembers'
+    defaultMessage='Members'
+    description='show opportunities list on volunteer home page'
+  />
+
+const orgInstructionTab =
+  <FormattedMessage
+    id='orgInstructions'
+    defaultMessage='Getting Started'
+    description='show opportunities list on volunteer home page'
+  />
+
+const orgOffersTab =
```

Comment:
no need to do this now - but see OpStatus for a more compact way to define a set of translated strings.